qa/suites/perf-basic/ceph.yaml

```diff
@@ -12,6 +12,7 @@ roles:
 - [mon.a, mgr.x, osd.0, osd.1, osd.2, client.0]
 tasks:
 - install:
+- ssh-keys:
```

Comment:
There is `ssh-keys:` already in this yaml. Do you mean to reorder it?

Reply:
sorry I missed that, I dont think it is the issue now, I will update here once I debug the failure in my testbed.

Reply:
The first issue seems to be pdsh/collectl which are available only in epel and we do use epel in octo lab, I can probably add that as an exception for this test.

2018-06-15T18:56:52.655 INFO:teuthology.orchestra.run.bruuni010.stdout:No package pdsh available.
2018-06-15T18:56:53.444 INFO:teuthology.orchestra.run.bruuni010.stdout:No package collectl available.